libraries/chain/include/graphene/chain/protocol/types.hpp

```diff
@@ -158,7 +158,6 @@ namespace graphene { namespace chain {
       impl_blinded_balance_object_type,
       impl_chain_property_object_type,
       impl_witness_schedule_object_type,
-      impl_budget_record_object_type,
```

Comment:
Removing this will change object IDs of subsequent types, i. e. fba_accumulator_objects and collateral_bid_objects. Must bump GRAPHENE_CURRENT_DB_VERSION in config.ini.

Since these are implementation objects, external applications SHOULD NOT rely on the IDs, so it SHOULD be ok to simply remove budget_record_object_type from the list. Thoughts @abitmore ?

Reply:
I think it's better to reserve the type ID like this: https://github.com/bitshares/bitshares-core/blob/9a4e745052fa7a209a44bad4affd30923918ad6e/libraries/chain/include/graphene/chain/protocol/types.hpp#L150, otherwise we may need to update lots of documents E.G. https://docs.bitshares.org/development/blockchain/objects.html

Reply:
added reserve at https://github.com/bitshares/bitshares-core/pull/1213/commits/e7c95b3b4be2ec2d77080e61622321f6b72eeb84